cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1294,9 +1294,10 @@
 
   <property>
     <name>messaging.system.topics</name>
-    <value>${audit.topic},${notification.topic}</value>
+    <value>${audit.topic},${metrics.topic.prefix}:${metrics.messaging.partition.num},${notification.topic}</value>
     <description>
-      A comma-separated list of topics that is always available in the system namespace
+      A comma-separated list of topics that is always available in the system namespace.
+      Topic prefix and topic partition are seperated by ":".
```

Comment:
TMS itself doesn't really have the "partition" concept. It's better not to use this terminology. You can say multiple topics will be created sequentially from 0 to n-1, prefixed with the given topic prefix.